src/main/java/com/kodilla/ecommercee/mapper/UserMapper.java

```diff
@@ -0,0 +1,38 @@
+package com.kodilla.ecommercee.mapper;
+import com.kodilla.ecommercee.domain.User;
+import com.kodilla.ecommercee.dto.UserDto;
+import org.springframework.stereotype.Component;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class UserMapper {
+
+    public User mapToUser(UserDto userDto) {
+        return new User(
+                userDto.getId(),
+                userDto.getUsername(),
+                userDto.getStatus(),
+                userDto.getUserKey()
```

Comment:
Wiem że UserDto ktoś już wrzucił  ale pytanie czy 
userKey to pole które na pewno chcemy mapować do DTO?  Czy te pole nie powinno być tajne?  Nie wiem do czego służy to pole, jeśli do logowania czy sesji to chyba nie powinno to się pojawić w jsonie

Reply:
Zgadzam sie z Toba, ale szczerze mowiac nie wiem jak bedzie w przyszlosci wygladac autoryzacji uzytkownika. Zawasze mozna pozniej dodac @JsonIgnore. Zastanawiam sie, czy poki co tego nie zostawic i zobaczyc jak to wyjdzie w praniu. W mapowaniu jest to potrzebne poniewaz musimy wyslac jakos nowy klucz do bazy danych, jedyne co to mysle ze UserDto mozna pozniej zaktualizowac.

Reply:
userKey chyba bedzie raczej na stale generowany tylko sessionId z klasy Login bedzie tymczasowy

Reply:
trzeba by tu poznać bardziej spring security żeby się wypowiedzieć  jak działają te mechanizmy.
Oczywiście zostawmy to tak jak jest na ten moment.   Chciałem tylko zwrócić uwagę na te pole.